agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4759,7 +4760,28 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
     }
 
     LOG(DEBUG) << "sending ACK message back to controller";
-    return send_cmdu_to_controller(cmdu_tx);
+    if (!send_cmdu_to_controller(cmdu_tx)) {
+        LOG(ERROR) << "failed to send ACK_MESSAGE";
+        return false;
+    }
+    //TODO - Send BTM Request frame to the STA including a Neighbor Report element specifying
+    // the BSSID, Operating Class and Channel Number of the identified target BSS
+
+    // build and send Client Steering BTM report message
+    if (!cmdu_tx.create(0, ieee1905_1::eMessageType::CLIENT_STEERING_BTM_REPORT_MESSAGE)) {
+        LOG(ERROR) << "cmdu creation of type CLIENT_STEERING_BTM_REPORT_MESSAGE, has failed";
+        return false;
+    }
+
+    auto seering_btm_report_tlv = cmdu_tx.addClass<wfa_map::tlvSteeringBTMReport>();
+    if (!error_code_tlv) {
```

Comment:
you check the wrong tlv (guess it is a copy-paste error)

Reply:
you are right, thanks!